K8s/helm/templates/cart-deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
           containerPort: 8080
         env:
           - name: OTEL_EXPORTER_JAEGER_ENDPOINT
-            value: "http://jaeger.monitoring.svc.cluster.local:14268/api/traces"
+            value: "http://otel-collector:14268/api/traces"
```

Comment:
Should this url be fully qualified? Also, should the collector also be in monitoring namespace?

Reply:
since it is in the same namespace it doesn't need to be, but if we are moving to monitoring namespace we need to change it a fully qualified name.
we have one collector for 3 containers, I tried to simulate the node level collector.

Reply:
Then, it's fine

Reply:
IMHO the collector should be in a separate namespace.